test/unittests/commands/validate_data/fixtures_validation_result.py

```diff
@@ -0,0 +1,207 @@
+@pytest.fixture
+def graph_value_type_mismatch() -> Graph:
+    gstr = f'''{VALIDATION_PREFIXES}
+    [] a sh:ValidationReport ;
+    sh:result _:bn_value_type_mismatch .
+
+    [] a sh:ValidationReport ;
+    sh:conforms false ;
+    sh:result [ 
+            a sh:ValidationResult ;
+            sh:detail _:bn_value_type_mismatch ;
+            sh:focusNode <http://data/value_type_mismatch> ;
+            sh:resultMessage """Value does not have shape
+                 <http://api.knora.org/ontology/knora-api/shapes/v2#ColorValue_ClassShape>""" ;
+            sh:resultPath onto:testColor ;
+            sh:resultSeverity sh:Violation ;
+            sh:sourceConstraintComponent sh:NodeConstraintComponent ;
+            sh:sourceShape onto:testColor_PropShape ;
+            sh:value <http://data/value-iri-value_type_mismatch> 
+            ] .
+
+    _:bn_value_type_mismatch a sh:ValidationResult ;
+    sh:focusNode <http://data/value-iri> ;
+    sh:resultMessage "ColorValue" ;
+    sh:resultSeverity sh:Violation ;
+    sh:sourceConstraintComponent sh:ClassConstraintComponent ;
+    sh:sourceShape <http://api.knora.org/ontology/knora-api/shapes/v2#ColorValue_ClassShape> ;
+    sh:value <http://data/value-iri-value_type_mismatch> .
+    '''
+    g = Graph()
+    g.parse(data=gstr, format="ttl")
+    return g
+
+
+@pytest.fixture
+def data_value_type_mismatch() -> Graph:
+    gstr = """
+    <http://data/value_type_mismatch> a <http://0.0.0.0:3333/ontology/9999/onto/v2#ClassWithEverything> .
+    <http://data/value-iri-value_type_mismatch> a <http://api.knora.org/ontology/knora-api/v2#TextValue> .
+    """
+    g = Graph()
+    g.parse(data=gstr, format="ttl")
+    return g
+
+
+@pytest.fixture
+def graph_wrong_regex_content() -> Graph:
+    gstr = f'''{VALIDATION_PREFIXES}
+    [] a sh:ValidationReport ;
+    sh:result _:bn_geoname_not_number .
+
+    [] a sh:ValidationReport ;
+    sh:conforms false ;
+    sh:result [ a sh:ValidationResult ;
+            sh:detail _:bn_geoname_not_number ;
+            sh:focusNode <http://data/geoname_not_number> ;
+            sh:resultMessage """Value does not have shape 
+                                        <http://api.knora.org/ontology/knora-api/shapes/v2#GeonameValue_ClassShape>""" ;
+            sh:resultPath onto:testGeoname ;
+            sh:resultSeverity sh:Violation ;
+            sh:sourceConstraintComponent sh:NodeConstraintComponent ;
+            sh:sourceShape onto:testGeoname_PropShape ;
+            sh:value <http://data/value-iri-geoname_not_number> ] .
+
+    _:bn_geoname_not_number a sh:ValidationResult ;
+    sh:focusNode <http://data/value-iri-geoname_not_number> ;
+    sh:resultMessage "The value must be a valid geoname code" ;
+    sh:resultPath knora-api:geonameValueAsGeonameCode ;
+    sh:resultSeverity sh:Violation ;
+    sh:sourceConstraintComponent sh:PatternConstraintComponent ;
+    sh:sourceShape api-shapes:geonameValueAsGeonameCode_Shape ;
+    sh:value "llllllllllllllllll" .
```

Comment:
*question* Is this the intended value?

This seems odd to me. I understand that this is "just" test data, but something more realistic would have not made me think. How about "this-is-not-a-valid-code"?

This comment also applies to the other parts of the code where you have used only `l`s.

Reply:
That is a good point.